lib/animina_web/components/chat/chat_components.ex

```diff
@@ -33,10 +34,19 @@ defmodule AniminaWeb.ChatComponents do
     ~H"""
     <div class="h-[80vh]  w-[100%] ">
       <div class="w-[100%] h-[100%] flex flex-col justify-between">
-        <div class="h-[7%] w-[100%]">
-          <.receiver_profile_box receiver={@receiver} />
+        <div class="h-[10%] w-[100%] z-50">
+          <ProfileComponents.profile_details
+            user={@receiver}
+            current_user={@sender}
+            current_user_has_liked_profile?={@current_user_has_liked_profile?}
+            profile_points={@profile_points}
```

Comment:
This is over bloated. It should be something like:

`<.ProfileComponents.profile_details user={@receiver} current_user={@sender} />`

We digg our own grave with the current approach. The user resource should auto load the flags, the points, etc. No need to have a million arguments for this which could all be part of `user`.

Reply:
Okay , let me take time to refactor the same thing , same for the profile_live , that should also close #262

Reply:
Let's merge it now and open a new one tomorrow. Done is better than perfect.